share/locale/en.json

```diff
@@ -1,165 +1,165 @@
-    "help_shutdown_timeout": "Shut down the onion service after N seconds",
-    "help_stealth": "Create stealth onion service (advanced)",
+    "help_stay_open": "Keep onion service running after OnionShare is completed",
+    "help_shutdown_timeout": "Close onion service after a given amount of seconds",
+    "help_stealth": "Make onion service stealthy (advanced)",
     "help_receive": "Receive files instead of sending them",
-    "help_debug": "Log application errors to stdout, and log web errors to disk",
+    "help_debug": "Log OnionShare errors to stdout, and log web errors to disk",
     "help_filename": "List of files or folders to share",
-    "help_config": "Path to a custom JSON config file (optional)",
-    "gui_drag_and_drop": "Drag and drop files and folders\nto start sharing",
+    "help_config": "Location of custom JSON config file (optional)",
+    "gui_drag_and_drop": "Drag and drop files and folders\nto start OnionSharing",
     "gui_add": "Add",
     "gui_delete": "Delete",
     "gui_choose_items": "Choose",
-    "gui_start_server": "Start Sharing",
-    "gui_stop_server": "Stop Sharing",
-    "gui_stop_server_shutdown_timeout": "Stop Sharing ({}s remaining)",
-    "gui_stop_server_shutdown_timeout_tooltip": "Share will expire automatically at {}",
-    "gui_copy_url": "Copy Address",
+    "gui_start_server": "Start OnionSharing",
+    "gui_stop_server": "Stop OnionSharing",
+    "gui_stop_server_shutdown_timeout": "Stop OnionSharing ({}s remaining)",
+    "gui_stop_server_shutdown_timeout_tooltip": "OnionShare expires at {}",
+    "gui_copy_url": "Copy OnionShare Address",
     "gui_copy_hidservauth": "Copy HidServAuth",
-    "gui_downloads": "Download History",
-    "gui_downloads_window_tooltip": "Show/hide downloads",
+    "gui_downloads": "File History",
+    "gui_downloads_window_tooltip": "Show/hide OnionSharings",
     "gui_no_downloads": "No downloads yet.",
     "gui_canceled": "Canceled",
-    "gui_copied_url_title": "Copied OnionShare address",
-    "gui_copied_url": "The OnionShare address has been copied to clipboard",
-    "gui_copied_hidservauth_title": "Copied HidServAuth",
-    "gui_copied_hidservauth": "The HidServAuth line has been copied to clipboard",
+    "gui_copied_url_title": "Copied",
+    "gui_copied_url": "OnionShare address copied to clipboard",
+    "gui_copied_hidservauth_title": "Copied",
+    "gui_copied_hidservauth": "HidServAuth line copied to clipboard",
     "gui_starting_server1": "Starting Tor onion service…",
     "gui_starting_server2": "Compressing files…",
     "gui_please_wait": "Starting… Click to cancel",
     "error_hs_dir_cannot_create": "Cannot create onion service dir {0:s}",
-    "error_hs_dir_not_writable": "onion service dir {0:s} is not writable",
+    "error_hs_dir_not_writable": "Onion service dir {0:s} is write protected",
     "using_ephemeral": "Starting ephemeral Tor onion service and awaiting publication",
```

Comment:
"Ephemeral" is better avoided or changed, though correct.

Reply:
You're right. This is a relic from a long time ago when OnionShare supported both the old kind of onion services and also ephemeral onion services. But now it only uses ephemeral ones, so I actually just completely removed this string in #779